src/implementations/twig/components/menu/menu.html.twig

```diff
@@ -183,10 +183,11 @@
       
       <ul class="ecl-menu__list" data-ecl-menu-list>
       {% if items is defined and items is not empty and items is iterable %}
-        {% for item in items %}
+        {% for key, item in items %}
         {% if item is defined and item is not empty %}
           {% include '@ecl/menu/menu-item.html.twig' with {
             item: item,
+            id: "ecl-menu-item-" ~ key,
```

Comment:
mmh..but what happens if we have two menus in a page? I understand that this might not be recommended or expected, but i think we should play safe adding a randomized string here